DevCycle.SDK.Server.Local/Api/LocalBucketing.cs

```diff
@@ -244,5 +270,17 @@ private static string ReadAssemblyScriptString(IStore store, Memory memory, int
             var length = memory.ReadInt32(address - 4);
             return Encoding.Unicode.GetString(memory.GetSpan(address, length));
         }
+
+        private void HandleError()
+        {
+            if (wasmError.Length > 0)
+            {
+                var error = new LocalBucketingException(wasmError);
+                WasmMutex.Release();
+                throw error;
```

Comment:
This would cause the stacktrace to be a bit confusing; maybe we want this (albiet very copy pasty) inside each call?

Reply:
wouldnt it just add one extra line?

Reply:
it'd just make it confusing because the initial stack is coming from some random "handleerror" function instead of a specific one. 
You could just return the exception and then throw if it exists from each function

Reply:
it's fine ill just duplicate it